PowerEditor/src/ScintillaComponent/FindReplaceDlg.h

```diff
@@ -165,6 +167,7 @@ protected :
 	bool _purgeBeforeEverySearch = false;
 
 	generic_string _prefixLineStr;
+	generic_string _LineNumFormatStr;

```

Comment:
The variable name starts always with lowercase. Please check contribution guide line for the coding convention of this project.